plugins/MauticCrmBundle/Integration/SalesforceIntegration.php

```diff
@@ -634,10 +649,15 @@ public function pushLead($lead,  $config = [])
 
                 foreach (['Contact', 'Lead'] as $object) {
                     if (!empty($existingPersons[$object])) {
+                        $fieldsToUpdate = $mappedData[$object]['update'];
+                        //check if update blank fields is selected
+                        if (isset($config['updateBlanks']) && isset($config['updateBlanks'][0]) && $config['updateBlanks'][0] == 'updateBlanks') {
```

Comment:
Can we update getBlankFieldsToUpdate to accept $config so that we can also move this line into the method itself. It'd be easy to mistype this if statement causing a mismatch in behavior. Also getBlankFieldsToUpdate may be good for CrmAbstract for others to use.